src/containers/CharacterProfile/MoveList.js

```diff
@@ -40,35 +40,33 @@ class MoveList extends Component {
     this.props.navigation.navigate('attackDetails', {move, index});
   }
 
-  renderByOrientation(orientation, moves) {
-    if (orientation == 'landscape') {
-      return (
-        <Spreadsheet moves={moves} onMovePress={(move, index) => this.onMovePress(move, index)} />
-      );
-    } else {
-      // plain list of moves
+  renderByOrientation(isPortrait, moves) {
       return (
+        <View>
+        <Spreadsheet isPortrait={isPortrait} moves={moves} onMovePress={(move, index) => this.onMovePress(move, index)} />
         <LinearGradient
           start={{x: 1.8, y: 0.4}} end={{x: 0.1, y: 0.9}}
           colors={[Colors.redSecondary, Colors.redPrimary]}>
             <FlatList
+            style={isPortrait ? {} : {display: 'none'}}
```

Comment:
For future reference, might be good to make a `Styles.hidden` variable for cleaner appraoch than show a hard-coded styles object